djstripe/event_handlers.py

```diff
@@ -31,90 +31,82 @@ def customer_event_attach(event, event_data, event_type, event_subtype):
     else:
         customer_stripe_id = event_data["object"].get("customer", None)
 
-    if not customer_stripe_id:
-        return
-
-    try:
-        event.customer = Customer.objects.get(stripe_id=customer_stripe_id)
-    except Customer.DoesNotExist:
-        pass
+    if customer_stripe_id:
+        try:
+            event.customer = Customer.objects.get(stripe_id=customer_stripe_id)
+        except Customer.DoesNotExist:
+            pass
 
 
 @webhooks.handler("customer")
-def customer_webhook_handler(
-        event, event_data, event_type, event_subtype):
+def customer_webhook_handler(event, event_data, event_type, event_subtype):
     """ Handles updates for customer objects. """
-    crud_type = CrudType.determine(event_subtype, exact=True)
-    if not crud_type.valid:
-        return
 
-    if not event.customer:
+    crud_type = CrudType.determine(event_subtype, exact=True)
+    if crud_type.valid and event.customer:
         # As customers are tied to local users, djstripe will not create
         # customers that do not already exist locally.
-        return
-
-    if crud_type.deleted:
-        # Deletions for customers are handled a little bit differently, since
-        # the customer is "purged" but not fully deleted from the system, so
-        # the object is updated first then "deleted".
-        _handle_crud_type_event(
-            Customer, event_data, event_subtype,
-            crud_type=CrudType(updated=True))
+        if crud_type.deleted:
+            # Deletions for customers are handled a little bit differently, since
+            # the customer is "purged" but not fully deleted from the system, so
+            # the object is updated first then "deleted".
+            _handle_crud_type_event(target_cls=Customer, event_data=event_data, event_subtype=event_subtype, crud_type=CrudType(updated=True))
 
-    _handle_crud_type_event(
-        Customer, event_data, event_subtype, crud_type=crud_type)
+        _handle_crud_type_event(target_cls=Customer, event_data=event_data, event_subtype=event_subtype, crud_type=crud_type)
```

Comment:
It seems like this could be made a bit DRYer:
```python
if crud_type.valid and event.customer:
    if crud_type.deleted:
        crud_type = CrudType(updated=True)

    _handle_crud_type_event(target_cls=Customer, event_data=event_data, event_subtype=event_subtype, crud_type=crud_type)
```